### PR TITLE
Isis adj tests

### PR DESCRIPTION
**Pull request by cprabha**
Hi,

I have created changes as per https://github.com/openconfig/public/pull/710.
please could you review and approve.

Thanks,
Prabha

### CODE REVIEW

File: feature/experimental/isis/ate_tests/base_adjacencies_test/base_adjacencies_test.go [hunk 1]
```diff
@@ -37,7 +37,7 @@ func TestMain(m *testing.M) {
 
 const (
 	PTISIS   = telemetry.PolicyTypes_INSTALL_PROTOCOL_TYPE_ISIS
-	ISISName = "osiris"
+	ISISName = "DEFAULT"
```
Comment (Ankur19) [inline feature/experimental/isis/ate_tests/base_adjacencies_test/base_adjacencies_test.go:40]: Remove the constant from this file and use `session.ISISName` where necessary.

**Review by Ankur19 — CHANGES_REQUESTED** [review]
Just needs a tiny change!

File: feature/experimental/isis/ate_tests/base_adjacencies_test/base_adjacencies_test.go (hunk 1 above)
Comment (cprabha) [reply feature/experimental/isis/ate_tests/base_adjacencies_test/base_adjacencies_test.go]: updated. thank you

**Review by Ankur19 — APPROVED** [review]
LGTM!

**Comment by Ankur19** [comment]
@cprabha You have to rebase with `main` for all checks to pass.

**Comment by cprabha** [comment]
> 

Thank you @Ankur19 .
 I have created new PR, please check .
I will close this one.

**Comment by Ankur19** [comment]
Closed in favour of #549

**Comment by cprabha** [comment]
Thank you.
-Prabha



Juniper Business Use Only

**Pull request closed without merging**